### PR TITLE
added .DS_store to gitignore

### PR DESCRIPTION
**Pull request by ankushduacodes**
its a mac system file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,6 @@
 .tox
 dist/
 wideq_state.json
-__pycache__
\ No newline at end of file
+__pycache__
+# Ignore Mac system files
+.DS_store
```
Comment (sampsyo) [inline .gitignore:7]:
```suggestion
.DS_Store
```

I think it's capitalized.

Comment (ankushduacodes) [reply .gitignore:7]: let me see

Comment (ankushduacodes) [reply .gitignore:7]: yes my bad

**Comment by sampsyo** [comment]
Thanks!!

**Pull request merged**